docs/mofed-container-env-vars.md

```diff
@@ -23,6 +23,7 @@ In most deployments its expected to not require setting those.
 | UNLOAD_STORAGE_MODULES |N|`"false"`| unload host storage modules prior to loading mofed modules  |
 | ENABLE_NFSRDMA |N|`"false"`| enable loading of nfs relates storage modules from mofed container|
 | RESTORE_DRIVER_ON_POD_TERMINATION |N|`"true"`| restore host drivers when container is gracefully stopped |
+| USE_DRIVERS_INVENTORY | N | `"true"` | enable use of a persistent directory to store drivers' build artifacts to avoid recompilation between runs |
```

Comment:
why wouldnt we want to direclty document: NVIDIA_NIC_DRIVERS_INVENTORY_PATH ?
if empty we can say that that build artifacts will not be stored and recompile will always happen

lets hear others opinions on this

Reply:
the rest of the values here are mapped directly to the container right ?

Reply:
Yes, I've changed it to use directly the `NVIDIA_NIC_DRIVERS_INVENTORY_PATH` env var